crates/bevy_core_pipeline/src/core_2d/mod.rs

```diff
@@ -127,8 +125,13 @@ impl Plugin for Core2dPlugin {
 
 /// Opaque 2D [`BinnedPhaseItem`]s.
 pub struct Opaque2d {
+    /// Determines which objects can be placed into a *batch set*.
+    ///
+    /// Objects in a single batch set can potentially be multi-drawn together,
+    /// if it's enabled and the current platform supports it.
+    pub batch_set_key: (),
```

Comment:
I don't understand the point of this unit type as a key. Is there more information on this pattern and what it's doing?

Reply:
It's just a placeholder because we need to put some type there. `DrawMesh2d` doesn't multi-draw, so the batch set key is irrelevant.